source/fundamentals/authentication.txt

```diff
@@ -25,10 +25,10 @@ of {+mongo-community+}:
 - :ref:`csharp-scram-sha-1`
 - :ref:`csharp-x509`
 
-To authenticate using ``Kerberos`` or ``LDAP``, see the
+To authenticate using ``GSSAPI/Kerberos`` or ``LDAP``, see the
 :ref:`csharp-enterprise-authentication-mechanisms` guide. For more information
```

Comment:
Suggestion: This link says "guide" and the next is "Guide" (because it's part of the page name). Maybe call the first one the "Authentication Mechanisms fundamentals page" or something similar?